haas/ext/auth/database.py

```diff
@@ -131,6 +132,19 @@ def user_remove_project(user, project):
     db.session.commit()
 
 
+@rest_call('PATCH', '/auth/basic/user/<user>', Schema({
+    'user': basestring,
+    'is_admin': bool,
+}))
+def user_set_admin(user, is_admin):
+    get_auth_backend().require_admin()
+    user = api._must_find(User, user)
+    if user.label == os.environ.get('HAAS_USERNAME'):
```

Comment:
I ran into an issue here because I didn't set the env variable where I was running the api server (`haas serve 5000`).

A user would set *his* environment variables. When you do an `os.environ.get` here it would return HAAS_USERNAME of the env where the server is running. So this mechanism would fail.

anybody know what's the right way to get the user here? from the URL?